ql/src/java/org/apache/hadoop/hive/ql/session/SessionState.java

```diff
@@ -270,6 +271,8 @@
 
   private SparkSession sparkSession;
 
+  private Exec hplsqlInterpreter;
```

Comment:
I don't fully understand why we have this field "here" - we don't even use hplsql from ql.

This field is only used from the "service" module

Reply:
I made this dynamic.